javascript-conditionals/main.js

```diff
@@ -0,0 +1,85 @@
+/* exported isUnderFive,
+            isEven,
+            startsWithJ,
+            isOldEnoughToDrink,
+            isOldEnoughToDrive,
+            isOldEnoughToDrinkAndDrive,
+            categorizeAcidity,
+            introduceWarnerBro,
+            recommendMovie
+ */
+function isUnderFive(number) {
+  if (number < 5) {
+    return true;
+  } else {
+    return false;
+  }
+}
+
+function isEven(number) {
+  if ((number % 2) === 0) {
+    return true;
+  } else {
+    return false;
+  }
+}
+
+function startsWithJ(string) {
+  return (string[0] === 'J');
+}
+
+function isOldEnoughToDrink(person) {
+  return (person.age >= 21);
+}
+
+function isOldEnoughToDrive(person) {
+  return (person.age >= 16);
+}
+
+function isOldEnoughToDrinkAndDrive(person) {
+  return false;
+}
+
+function categorizeAcidity(pH) {
+  if (pH < 7 && pH >= 0) {
+    return 'acid';
+  } else if (pH === 7) {
+    return 'neutral';
+  } else if (pH > 7 && pH <= 14) {
+    return 'base';
+  } else {
+    return 'invalid pH level';
+  }
+}
+
+function introduceWarnerBro(name) {
+  switch (name.toLowerCase()) {
+    case 'yakko':
+      return 'We\'re the warner brothers!';
+    case 'wakko':
+      return 'We\'re the warner brothers!';
+    case 'dot':
+      return 'I\'m cute~';
+    default:
+      return 'Goodnight everybody!';
+  }
+}
+
+function recommendMovie(genre) {
+  switch (genre) {
+    case 'action':
+      return 'Die Hard';
+    case 'comedy':
+      return 'The Big Lebowski';
+    case 'horror':
+      return 'The Ring';
+    case 'drama':
+      return 'Awaken';
+    case 'musical':
+      return 'High School Musical';
+    case 'sci-fi':
+      return 'The Mandalorian';
+    default:
+      return 'try again bud';
```

Comment:
Funny, but we want to inform the user where they went wrong if they passed an incorrect value! Take a look at the error message I gave, and use the same one.

Reply:
updated!